server/web/edit/index.js

```diff
@@ -64,7 +65,11 @@ exports.register = function (server, options, next) {
         }
         let page = Hoek.applyToDefaults(defaults.testPageContext, request.payload, true);
         Hoek.merge(page, errObj);
-        reply.view('edit/index', {page: page, authorized: true}).code(400);
+        reply.view('edit/index', {
+          headTitle: page.title,
```

Comment:
can you add a test for the edit page similar to test page? would be nice if it's verified for both 200 and 400 scenarios